swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=0.0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
Nice job hard-coding the category here. In the future, consider using `super` as well. Even though it doesn't gain us a ton in this specific instance, it makes updating our code easier in the future if we ever add extra logic to the parent class that we want inherited by all the children.